modules/ROOT/pages/single-sign-on-with-oidc.adoc

```diff
@@ -232,7 +250,7 @@ You may also define some token signature and encryption mechanism. In this case,
 [NOTE]
 ====
 
-If the IdP declares a redirect/target URL, it might override the target URL set by the Service Provider request, and you may always end up on the same page after logging in. In that case, try to remove the redirect URL. Bonita supports redirection to the URL initially requested after logging in on the IdP, provided the IdP doesn't force this URL.
+If the IdP declares a redirect URI, it might override the target URL (`redirect_URI` parameter) set by the Bonita OIDC module in the request, and you may always end up on the same page after logging in. In that case, try to remove the redirect URI from your OIDC provider's Bonita client configuration. Bonita supports redirection to the URL initially requested after logging in on the IdP, provided the IdP doesn't force this URL.
```

Comment:
"IdP" -> whats IDP means ?

Reply:
Identity provider (it is the OIDC provider server). It is explained in the first sentence of this page

Reply:
but i will clarify it